internal/security/setup/option/optionHelpers.go

```
@@ -214,20 +233,64 @@ func markComplete(dirPath string) error {
 	return nil
 }
 
-func getXdgRuntimeDir() string {
+func getWorkDir() string {
```

as mentioned above, I think this function should also return an error

---

ok for that